src/webhook/validation/istio.go

```diff
@@ -11,11 +14,28 @@ const (
 
 func noResourcesAvailable(dv *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
 	if dynakube.Spec.EnableIstio {
-		enabled, err := istio.CheckIstioEnabled(dv.cfg)
+		enabled, err := CheckIstioInstalled(dv.cfg)
 		if !enabled || err != nil {
 			return errorNoResources
 		}
 	}
 
 	return ""
 }
+
+func CheckIstioInstalled(cfg *rest.Config) (bool, error) {
```

Comment:
this should be in the `istio` package

Reply:
unless you want to check if istio exists anywhere else, but it could be private if not used anywhere else

Reply:
Made the function private